src/containers/MyNdla/Arena/PostsPage.tsx

```diff
@@ -109,19 +78,27 @@ const PostsPage = () => {
         setFocusId(newReply.data.replyToTopicV2.id);
       }
     },
-    [replyToTopic, topicId, setFocusId],
+    [replyToTopic, arenaTopic?.id, setFocusId],
   );
 
-  const onFollowChange = useCallback(() => {
+  useEffect(() => {
+    if (!authContextLoaded || !user?.arenaEnabled || loading) return;
+    trackPageView({
+      title: t("htmlTitles.arenaPostPage", { name: arenaTopic?.title ?? "" }),
+      dimensions: getAllDimensions({ user }),
+    });
+  }, [arenaTopic?.title, authContextLoaded, loading, t, trackPageView, user]);
```

Comment:
Noen grunn til at vi flytter denne? Synes det blir rart å ha en effect mellom to funksjoner.